### PR TITLE
s/master/primary in agnhost guestbook usage

### PR DESCRIPTION
**Pull request by BenTheElder**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Follow up to https://github.com/kubernetes/kubernetes/pull/92119

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by BenTheElder** [comment]
https://github.com/kubernetes/k8s.io/pull/977

**Review by hasheddan — APPROVED** [review]
/lgtm

Thanks @BenTheElder!

ref: #89262 #88974

File: test/utils/image/manifest.go [hunk 1]
```diff
@@ -202,7 +202,7 @@ const (
 
 func initImageConfigs() map[int]Config {
 	configs := map[int]Config{}
-	configs[Agnhost] = Config{promoterE2eRegistry, "agnhost", "2.19"}
+	configs[Agnhost] = Config{promoterE2eRegistry, "agnhost", "2.20"}
```
Comment (BenTheElder) [inline test/utils/image/manifest.go:205]: this is where we pick up the new image with the service name change

**Pull request merged**